ntp.go

```diff
@@ -159,6 +162,87 @@ type QueryOptions struct {
 	TTL     int           // IP TTL to use for outgoing UDP packets, defaults to system default
 }
```

Comment:
Can you move the declaration of `QueryOptions` above the declaration of `Response`?  That way all your Response methods will be grouped with the `Response` declaration.